src/kata1.fizzBuzz.js

```diff
@@ -1,5 +1,20 @@
 const fizzBuzz = (number) => {
+    const isDivisableBy3 = (number % 3) === 0;
+    const isDivisableBy5 = (number % 5) === 0;
 
-}
+    if (isDivisableBy3 && isDivisableBy5) {
```

Comment:
nice use of variables to annotate intent, makes your function easier to read and follow 👍